automated-comments/javascript/gigasecond/dont_use_date_parse.md

```diff
@@ -1,7 +1,3 @@
-Use [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime),
-as `Date.parse(%<parameter.name>)` is not a good candidate.
+Your solution used `Date.parse(%<parameter.name>)`. This works, but isn't _intended_ to be used with a `Date` object. In fact, in order to work it first coerces the input value to a `string` and then parses it back into a date, which is unnecessary in this case.
 
-`Date.parse` is supposed to be called with a `string`, and therefore not
-_intended_ to be used like this. The reason it does work is because `Date.parse`
-first coerces the input value to a `string` and then parses it back into a date.
-These extra conversions are unnecessary.
+It would be better to use `%<parameter.name>.getTime()` as [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) doesn't coerce the input but works on the Date directly.
```

Comment:
```suggestion
It is better to use `%<parameter.name>.getTime()` as [`Date#getTime`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime) doesn't coerce the input but works on the Date directly.
```

Reply:
Gathered from @kotp suggestion.

Reply:
I prefer "would be" I think. "is" feels a lot stronger, "would be" feels a bit more chatty and friendly to me.

Reply:
I use "would be" in conjunction with "if something is done".  It "is better" because of a specific reason, as given here.